### ISSUE TITLE
[TW-99] info report streamlining

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#156]
_David Patrick on 2014-01-26T04:36:25Z says:_ 

 The info report is laid out logically, it has two columns: Name and Value. 
It is an essential report, and I'd like to see it easier to read and about 3 lines shorter. 

Firstly, the description is "buried" with the other attributes, in multiple-info reports the description need to be more obvious. Secondly, users don't need to be told what column is Name and what column is Value, that's superfluous. 

So taking these two things together, and adding a third column for urgency (see issue #1078) the new streamlined info might look like this:

<pre>
$ tw 90 info

90___________ pay_Teksavvy___________________________ 19.0                  
Status        Pending
Project       telcom                                   1.0                
Priority      M                                        6.0
Recurrence    monthly                             
Until         2014-05-15 16:29:42
Parent task   2322e1c5-604d-4310-b93c-e4687f6011a2
Mask Index    10
Due           2014-02-01 00:00:00                     12.0
UUID          c821cec4-1e44-480c-ab4d-eb97170ba238
Entered       2014-01-19 21:45:29 (6 days)        
Last modified 2013-12-15 00:32:00 (5 wks) 
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-01-26T04:36:25Z
Modified: 2014-08-31T20:24:01Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2014-01-26T11:16:04Z says:_ 

 Scott Kostyshak made a good suggestion (over IRC) that the displayed dates should perhaps be in a better grouping/ order, and I think he's right. While we're on it, it feels like the order in the info report is a bit random, so allow me to re-sketch it with dates grouped, recur and mask grouped.

<pre>
$ tw 90 info

90___________ pay_Teksavvy___________________________ 19.0                  
Status        Pending
Project       telcom                                   1.0                
Priority      M                                        6.0
Recurrence    monthly
Mask Index    10 
Entered       2014-01-19 21:45:29 (6 days)
Due           2014-02-01 00:00:00                     12.0                               
Until         2014-05-15 16:29:42
Parent task   2322e1c5-604d-4310-b93c-e4687f6011a2
UUID          c821cec4-1e44-480c-ab4d-eb97170ba238
Last modified 2013-12-15 00:32:00 (5 wks) 
</pre>

"Last Modified" is arguable, whether it should be grouped with the other dates, both it and "Until" (for a recurring task) refer to the parent task, so that they will be out of date-sequence and that fact is not entirely obvious from here, see issue #1090.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-01-26T18:23:49Z says:_ 

 I have reordered the dates to appear consecutively in this sequence:

* entry
* wait
* scheduled
* start
* due
* end
* until
* modified

I have not eliminated the table column headers (verbose=labels sort of does this), nor have I replaced "Description" with the ID.  The unit tests rely heavily on the structure of the info report.  Changing this would generate an enormous amount of work, and we have an enormous lack of volunteers:

<pre>
$ grep info task.git/test/* | wc -l
     212
</pre>

Instead I will gradually migrate the unit tests away from the info report, toward the export command, which is a much better solution.

I am leaving this ticket open, and mark it partially completed.
